cypress/integration/26-register_publication/creator_navigates_to_the_description_tab_and_handles_projects/handle_projects.js

```diff
@@ -0,0 +1,66 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+
+const filename = 'example.txt';
+
+Given('Creator begins registering a Registration in the Wizard', () => {
+  cy.login(USER_WITH_AUTHOR);
+  cy.startRegistrationWithFile(filename);
+  cy.get('[data-testid=registration-file-start-button]').click({ force: true });
+});
+When('they navigate to the Description tab', () => {
+  cy.get('[data-testid=nav-tabpanel-description]').click({ force: true });
+});
+And('they see a Search box for Projects', () => {
+  cy.get('[data-testid=project-search-input]').should('be.visible');
+});
+And('they enter search term in the Search box', () => {
+  cy.get('[data-testid=project-search-input]').type('a test battery for assessing pain');
+});
+Then('they see list of Projects matching the search term', () => {
+  cy.get('[class=MuiAutocomplete-option]').as('options');
+  cy.get('@options').then((options) => {
+    console.log(options);
+  });
+  cy.get('[data-testid^=project-option]').should('have.length.above', 0);
+});
+And('they see title and associated Institutions for each Project', () => {
+  cy.get('[data-testid^=project-option]').contains('a test battery for assessing pain');
+  cy.get('[data-testid^=project-option]').contains('Høgskulen på Vestlandet');
+});
+
+Given('Creator has searched for a Project', () => {
+  cy.login(USER_WITH_AUTHOR);
+  cy.startRegistrationWithFile(filename);
+  cy.get('[data-testid=registration-file-start-button]').click({ force: true });
+  cy.get('[data-testid=nav-tabpanel-description]').click({ force: true });
+});
+And('they see Search results', () => {
+  cy.get('[data-testid=project-search-input]').type('a test battery for assessing pain');
+});
+When('they select a Project from the Search results', () => {
+  cy.get('[class=MuiAutocomplete-option]').contains('a test battery for assessing pain').click({ force: true });
+});
+Then('the selected Project is added to the list of selected Projects', () => {
+  cy.get('[data-testid^=project-chip]').should('have.length', 1);
+  cy.get('[data-testid^=project-chip]').contains('test-retest av test batteri');
+  // TODO fix when project endpoint returns english project title

```

Comment:
ligger ute PR for dette nå

Reply:
skal være klart i dev nå (merk at ikke alle prosjekter har både norsk og engelsk tittel)